### PR TITLE
set _autoHide to default to false

### PR DESCRIPTION
**Pull request by moloko**
in order to match the changes made in #94

### CODE REVIEW

File: js/handlers/button.js [hunk 1]
```diff
@@ -41,48 +41,47 @@ define([
...
-                "_isDisabled": false
-            }, trickle._button);
-
+            trickle._button = _.extend(defaults, trickle._button);
 
             if (trickle._button._isFullWidth) {
                 trickle._stepLocking._isEnabled = true;
-                trickle._button._styleAfterClick = "hidden";
-            } else {
-                trickle._button._autoHide = false;
```
Comment (oliverfoster) [inline js/handlers/button.js]: Why did you remove this line?

Comment (moloko) [reply js/handlers/button.js]: because I didn't think it through properly ;-) I'll revert that change shortly.

Comment (oliverfoster) [reply js/handlers/button.js]: Cool :)

Comment (moloko) [reply js/handlers/button.js]: all done, thanks for catching that one!
